cmd/mobile-security-service/main.go

```diff
@@ -27,6 +28,8 @@ func main() {
 	// Load middleware
```

Comment:
we should also be defining an api Group prefix for these endpoints. The same as the MDC https://github.com/aerogear/mobile-developer-console/blob/f49ebbe75bbbcfcd853f95525ac3b663fb84e598/cmd/api-server/main.go#L46

Reply:
@damienomurchu is working on this in his routing solution

Reply:
@davidffrench Yes, this will be added in #16. That PR relies on changes made in this PR, but will be merged in after this one